lib/src/macos_app.dart

```diff
@@ -191,23 +317,21 @@ class _MacosAppState extends State<MacosApp> {
 
   Widget _builder(BuildContext context, Widget? child) {
     final theme = this.theme(context);
-    return m.Material(
-      child: MacosTheme(
-        style: theme,
-        child: DefaultTextStyle(
-          style: TextStyle(
-            color: theme.typography?.body?.color,
-          ),
-          child: child!,
+    return MacosTheme(
+      style: theme,
+      child: DefaultTextStyle(
+        style: TextStyle(
+          color: theme.typography?.body?.color,
         ),
+        child: child!,
       ),
     );
   }
 
   Widget _buildApp(BuildContext context) {
-    final defaultColor = widget.primaryColor ?? CupertinoColors.systemBlue;
+    final defaultColor = widget.color ?? CupertinoColors.systemBlue;
```

Comment:
While `WidgetsApp` and `CupertinoApp` do call this property `color`, I feel `primaryColor` is more accurate. That said, an even more accurate name would probably be `systemColor`, since the documentation for this property defines it as being the primary color used by the OS for the app.

Reply:
`primaryColor` is definitely better than `systemColor`, because the dev may think `sytemColor` is the color used by the system, not the app.

Reply:
Fair enough, let's go with `primaryColor`